### PR TITLE
Chore: update code and reports after audits

### PR DESCRIPTION
**Pull request by amusingaxl**

### CODE REVIEW

File: script/phase-1b/03-UsdsSkyFarmingCheck.s.sol [hunk 1]
```diff
@@ -54,6 +58,9 @@ contract Phase1b_UsdsSkyFarmingCheckScript is Script {
         require(DssVestWithGemLike(vest).res(vestId) == 1, "DssVest/invalid-vest-res");
         require(DssVestWithGemLike(vest).usr(vestId) == dist, "DssVest/wrong-dist");
         require(DssVestWithGemLike(vest).mgr(vestId) == address(0), "DssVest/mgr-should-not-be-set");
+        require(DssVestWithGemLike(vest).bgn(vestId) == vestBgn, "DssVest/invalid-bgn");
```
Comment (0xdecr1pto) [inline script/phase-1b/03-UsdsSkyFarmingCheck.s.sol:61]: You are missing chek for ` Vest.cap`

Comment (0xdecr1pto) [reply script/phase-1b/03-UsdsSkyFarmingCheck.s.sol:61]: Saw you add `file(CAP)` to the `UsdsSkyFarmingInitSpell `

Comment (amusingaxl) [reply script/phase-1b/03-UsdsSkyFarmingCheck.s.sol:61]: This is out of the scope of this repo.
Cap is a system parameter.

Comment (0xdecr1pto) [reply script/phase-1b/03-UsdsSkyFarmingCheck.s.sol:61]: Nvm, saw your comments on the issue

**Review by 0xdecr1pto — APPROVED** [review]
LGTM

**Review by oddaf — APPROVED** [review]

**Review by oddaf — APPROVED** [review]

**Pull request merged**